apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -195,9 +196,22 @@ API.v1.addRoute(
 	},
 	{
 		async post() {
-			const { rid, comment, tags, generateTranscriptPdf, transcriptEmail } = this.bodyParams;
+			const { rid, comment, tags, generateTranscriptPdf, transcriptEmail, forceClose } = this.bodyParams;
 
-			await closeLivechatRoom(this.user, rid, { comment, tags, generateTranscriptPdf, transcriptEmail });
+			const allowForceClose = rcSettings.get<boolean>('Omnichannel_allow_force_close_conversations');
```

Comment:
What if someone tries to force close a room in a workspace where this is not enabled? Shouldn't we throw an error? I wonder if just silently not force closing could be misleading

Reply:
It will throw an error. It will be the normal room closing error as of the setting was ignored.

Didn't want to add a new error just for this case, as it could be confusing